### PR TITLE
New label

### LINKED ISSUE #983: New Label Component
https://app.abstract.com/projects/23793d20-e68f-11e8-816f-b177d180e919/branches/a6aed27d-7547-4fdb-8dec-f053bd140871/commits/896d3b42f0a52f59513d9f8e62f0b4b48fbd4c82/files/3157643b-b558-4178-b524-1ae72afecf67/layers/5B5B6448-1BC1-4442-B63E-9C02B9861C2E?collectionId=1f5193f4-95dd-4a1d-b117-59d62d58fc68&collectionLayerId=694d2bb0-4fba-474a-a4f9-cd7766f44ead&mode=build&selected=root-5B5B6448-1BC1-4442-B63E-9C02B9861C2E

### PR DESCRIPTION
**Pull request by alcferreira**
## Closes Issues

* Closes #983

### Reviewer 1

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### Reviewer 2

* [ ] Functions properly
* [ ] Good readability, clarity and overall quality
* [ ] Has tests
* [ ] Has docs

### CODE REVIEW

File: packages/label/src/Label.js [hunk 1]
```diff
@@ -19,63 +15,21 @@ const labelColor = props => {
...
 };
 
-const TextLabel = styled(Text)`
-  order: 1;
-  margin-bottom: 8px;
-  float: left;
-`;
+const Label = styled.label`
+  font-family: Lato, sans-serif;
+  display: block;
```
Comment (vNakamura) [inline packages/label/src/Label.js:20]: By having the label as a block, it takes the whole available width.
It seems weird in the example, but I'm not sure if this is good or bad in real usage as the inputs would be larger 🤔.

File: packages/label/src/Label.story.js [hunk 2]
```diff
@@ -1,41 +1,66 @@
...
+};
+
+const Container = styled.div`
+  margin-bottom: 24px;
+`;
+
+export const Labels = () => (
+  <div>
+    <Container>
+      <Label for="default">Default Label</Label>
```
[the remark this answers is not in the document]
Comment (alcferreira) [reply packages/label/src/Label.story.js:17]: totally missed that

File: packages/label/src/Label.js (hunk 1 above)
Comment (alcferreira) [reply packages/label/src/Label.js:20]: Makes sense... It's probably a better idea to let hoc-input handle how it should be rendered

Comment (alcferreira) [reply packages/label/src/Label.js]: I'll fix the margin bottom on this pr https://github.com/CraveFood/farmblocks/pull/988

**Pull request merged**